corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js

```diff
@@ -705,6 +708,11 @@ cloudCare.AppView = Backbone.View.extend({
                             lineNumbers: true,
                         });
 
+                        $instanceTab.off();
```

Comment:
this is ugly, but in order to not put this here, we need to do a pretty invasive refactor the debugger so i punted (debugger should belong in `fullform-ui.js`)